conf/messages

```diff
@@ -1,9 +1,5 @@
 site.govuk = GOV.UK
 
-checkYourAnswers.title = Check Your Answers
```

Comment:
Were these meant to be removed?

Reply:
Yes they weren't being used anywhere. Everything works as expected